### ISSUE TITLE
Tooltip text announced by ScreenReader in different OS versions incorrectly

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#2538]
* .NET Core Version: 5.0.100-alpha1-015918
* Have you experienced this same bug with .NET Framework?: Yes

**Problem description:**
- Some users in different OS versions have a problem with a tooltip text announcing.

**Actual behavior:** 
- **RS5** OS version (2018): Narrator announces a tooltip text **once**. But after #2074 Narrator announces a text **twice**.
![image](https://user-images.githubusercontent.com/49272759/71243924-7068d480-2322-11ea-8404-975bf60d3211.png)

- **19H2** OS version (2019): Narrator doesn't announce a tooltip text. But after #2074 Narrator announces a text **once**.
![image](https://user-images.githubusercontent.com/49272759/71243523-90e45f00-2321-11ea-968a-917d25816e7a.png)

**Expected behavior:**
- ScreenReader should announce a tooltip text once.

**Minimal repro:**
- Create a Core 5.0 app
- Add a control to the app
- Add a tooltip text to the control
- Run Narrator and listen to what it says when showing a tooltip

**Comment by vladimir-krestov** [comment]
The related bug [987891](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/987891)
As was discussed: 
> "The user must use the Advanced Information hotkey (`"Narrator key + 0"`) to read the tooltip."

It is applicable for Narrator in OS versions later than v.1809 (RS5). This is a new rule for using Narrator.
In v.1809 and earlier Narrator just reads a tooltip text when a mouse hovers a control without using any combinations (`"Narrator key + 0"`).

I added the additional tooltip text announcing in #2074. And now we have 2 cases:
- RS5: Narrator announces a tooltip text + my added announcing = the text is announced twice (incorrect)
- 19H2: We see my added announcing only = the text is announced once (expected)

@merriemcgaw, I see 2 ways:
- Keep it as is and **close** the Issue. Narrator announcing when a mouse hovers a control is comfortable to me. And it's Ok for users with the latest OS versions.
- Revert #2074. Narrator will announce a tooltip text in **RS5** once as before, but users have to use `"Narrator key + 0"` combination to announce a tooltip text in the latest OS versions as required by the new rules for the use of Narrator.

@merriemcgaw, _which way should we choose?_

**Comment by merriemcgaw** [comment]
We should choose the new way! The only negative impact is announcing twice and that is better than not announcing at all. Our goal is to make WinForms as accessible as possible _on the latest Windows Version_

**Comment by vladimir-krestov** [comment]
In this case, I propose to close this issue and keep it as is. We will reopen this when we get a request from users.
@merriemcgaw, please close this issue if you agree.
